aws/table_aws_drs_job.go

```diff
@@ -0,0 +1,184 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+
+	"github.com/aws/aws-sdk-go-v2/service/drs"
+	"github.com/aws/aws-sdk-go-v2/service/drs/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+func tableAwsDRSJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_drs_job",
+		Description: "AWS DRS Job",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "job_id", Require: plugin.Optional},
+				{Name: "from_date", Require: plugin.Optional},
```

Comment:
Instead of adding new columns `from_date` and `to_date`, can we use `creation_date_time` and `end_date_time` values?

We need to manipulate the value as per the operator provided in the where clause.
For reference - https://github.com/turbot/steampipe-plugin-aws/pull/1437